src/main/java/org/traccar/web/shared/model/Sensor.java

```diff
@@ -105,6 +105,17 @@ public void setVisible(boolean visible) {
         this.visible = visible;
     }
 
+    private boolean onGraph;
```

Comment:
This should be defined `nullable` at least. Then there should be an migration, which will set up `false` for all existing reports. Otherwise database will be damaged and reports could not be opened.